package.json

```diff
@@ -50,11 +50,13 @@
     "stylelint": "^14.16.1",
     "stylelint-config-prettier-scss": "^0.0.1",
     "stylelint-config-recommended-scss": "^8.0.0",
+    "typescript": "^5.1.3",
     "wait-on": "^7.0.1"
   },
   "prettier": "@infoprojects/prettier-config",
-  "packageManager": "yarn@3.5.1",
+  "packageManager": "yarn@3.6.0",
   "resolutions": {
-    "@types/react@^16.14.23": "18.0.25"
+    "@types/react@^16.14.23": "18.0.25",
+    "puppeteer-core@^2.1.1": "13.5.2"
```

Comment:
`sb extract` maakt gebruik van Puppeteer.

`puppeteer-core` voorziet in een eigen browser, namelijk Chromium. Versie 2.1.1 werkt met Chromium 80, maar de code die Angular genereert is daar te modern voor. Met 13.5.2 installeren we Chromium 100. Wat ik overigens ook maar gewoon heb verzonnen.